vtex-apps/search-result
# Fix name being used directly as slug

#### What is the purpose of this pull request?
Parse the name before using it as a slug.

#### What problem is this solving?
The name of the facet was being used as a slug directly, and broke in the cases where the slug isn't the same as the name (e.g. `Make B.` as opposed to `make-b-`).

#### How should this be manually tested?
[Workspace](https://lucas--storecomponents.myvtex.com/cosmeticos/d), select the `Make B.` brand facet and you should be redirected correctly.

#### Types of changes
